source/Grid/Grid.js

```diff
@@ -63,6 +64,12 @@ export default class Grid extends Component {
     noContentRenderer: PropTypes.func.isRequired,
 
     /**
+     * Optional callback invoked on mouse-wheel activity.
+     * (event: SyntheticEvent): void
+     */
+    onWheel: PropTypes.func,
```

Comment:
> I think the grid _should_ listen to more of the html events.

Mind elaborating? Since relevant events (keyboard events, mouse events, etc.) bubble I didn't want to go down the path of having to pass-through all of the listeners. It didn't seem necessary.